android/defaults/deps.nix

```diff
@@ -450,7 +450,7 @@
   { artifactId = "apache";
     groupId = "org.apache";
     version = "13";
-    repo = "https://repo.maven.apache.org/maven2/";
+    repo = "https://repo1.maven.apache.org/maven2/";
```

Comment:
Should this be repo1.maven.org instead?

Reply:
yeah it should. Damn it. Thanks for the catch!